### PR TITLE
Request detail page

### LINKED ISSUE #219: US_1.23: Anfragen Details page
**As** 
organizer
 
**I want to** 
be able to click on an request from the request page and get the detail view. On that page i also want to be able to change the status from open to accepted, by clicking an "Anfrage annehmen" button

**in order to** 
get a detailed overview

**🍪**

### PR DESCRIPTION
**Pull request by MasterCarl**
Closes #219.

### CODE REVIEW

**Review by bjrne — CHANGES_REQUESTED** [review]
I miss the schema file(?). The rest looks good.

**Comment by MasterCarl** [comment]
@bjrne What do you mean? I ran `rake db:drop db:migrate` and saw no changes to `schema.rb`.

**Comment by bjrne** [comment]
@MasterCarl Weird. You created a migration that adds a column and still don't get a change in `schema.rb`? I don't know how that's possible. Do you have any idea @tom95?

**Comment by MasterCarl** [comment]
@bjrne Looks like the schema changes are already on the `dev` branch...

**Comment by bjrne** [comment]
@MasterCarl true, it was you 😂 ba289f0.

**Review by bjrne — APPROVED** [review]
Beside this small change, everything looks good 👍

File: spec/features/request_spec.rb [hunk 1]
```diff
@@ -58,4 +58,25 @@ def fill_in_required_fields
...
+      end
+
+      it "should allow me to change the status to :accepted" do
+        request = FactoryGirl.create(:request, status: :open)
+        visit(request_path(request))
+        status = I18n.t(@request.status, scope: 'activerecord.attributes.request.statuses')
+        expect(page).to have_text(status)
+        click_link I18n.t('requests.form.accept')
+        request = Request.find(request.id)
+        expect(request.status.to_sym).to eq(:accepted)
```
Comment (bjrne) [inline spec/features/request_spec.rb]: Tom pointed out that since this is a feature test, database access isn't really wanted here.

**Comment by bjrne** [comment]
@MasterCarl #336 removed the status line from the `schema.rb` in order to have the right schema. You should now be able (and forced) to add it here again.

**Pull request merged**